### PR TITLE
Relax packaging and xxhash requirements

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: changes/61a5ed1d20140ec69aea3b7931355441.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+desc: Update the allowed versionf of the ``packaging`` and ``xxhash`` libraries.
```
Comment (MichaelSquires) [inline changes/61a5ed1d20140ec69aea3b7931355441.yaml:2]:
```suggestion
desc: Update the allowed version of the ``packaging`` and ``xxhash`` libraries.
```

Comment (mikemoritz) [inline changes/61a5ed1d20140ec69aea3b7931355441.yaml:2]:
```suggestion
desc: Update the allowed version of the ``packaging`` and ``xxhash`` libraries.
```

Comment (vEpiphyte) [reply changes/61a5ed1d20140ec69aea3b7931355441.yaml]: it only lets me apply the change once. @MichaelSquires won the race :laughing: :racing_car:

Comment (MichaelSquires) [reply changes/61a5ed1d20140ec69aea3b7931355441.yaml]: :racehorse:

**Review by MichaelSquires — APPROVED** [review]

**Pull request merged**